en/companion_computer/holybro_pixhawk_rpi_cm4_baseboard.md

```diff
@@ -33,17 +31,42 @@ It can therefore be used with other boards that follow these standards, but may
 
 ## Connections & Ports
 
+:::note
+The [Holybro Documentation](https://docs.holybro.com/autopilot/pixhawk-baseboards/pixhawk-rpi-cm4-baseboard/connections-and-ports) has more detailed (and possibly more "up to date") port and connection information.
+:::
+
 The diagram below shows all the connectors and ports on the baseboard.
 
 ![Schematic diagram](../../assets/companion_computer/holybro_pixhawk_rpi_cm4_baseboard/baseboard_ports.jpg)
 
-Refer to the [Holybro Documentation page](https://docs.holybro.com/autopilot/pixhawk-baseboards/pixhawk-rpi-cm4-baseboard/connections-and-ports) for the latest infofmation.
+### RPi CM4 & FC Serial Connection
+
+The flight controller `TELEM2` port is internally connected to RPi CM4 as shown: 
+
+RPi CM4 | FC TELEM2 (FMU)
+--- | ---
+GPIO14 | TXD
+GPIO15 | RXD
+GPIO16 | CTS
+GPIO17 | RTS
+
+:::note
+The connection must be also be [configured in both RPi and PX4](#configure-px4-to-cm4-mavlink-serial-connection) (unless an [Ethernet](#ethernet-connection) is used instead).
+:::
+
+## Installing the Flight Controller
+
+A plug-compatible flight controller such as [Holybro Pixhawk 5X](../flight_controller/pixhawk5x.html) and [Holybro Pixhawk 6X](../flight_controller/pixhawk6x.html) can simply be pushed into the module slot.
+
+Flight controllers that have a different form factor will need additional wiring.
```

Comment:
Is this true?

Reply:
If it follow Pixhawk Autopilot Bus, then it should be plug and play.

Reply:
@mrpollo continue to work with the FC manufacturer before the FC "Pixhawk certified". One of the criteria is that the dimension & position of the module. If they follow the criteria, then there should be enough space to plug and play.

Reply:
Ah, thanks. The mechanical guidelines section are _recommendations_. I'll see if this can be made more precise.